fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -620,10 +621,25 @@ public void testGetTimeMapResponseForBinaryDescription() throws Exception {
         verifyTimemapResponse(descriptionUri, descriptionId);
     }
 
+    /**
+     * Verify an application/link-format TimeMap response.
+     *
+     * @param uri The full URI of the Original Resource.
+     * @param id The path of the Original Resource.
+     * @throws Exception
+     */
     private void verifyTimemapResponse(final String uri, final String id) throws Exception {
         verifyTimemapResponse(uri, id, null, null, null);
     }
 
+    /**
+     * Verify an application/link-format TimeMap response.
+     *
+     * @param uri The full URI of the Original Resource.
+     * @param id The path of the Original Resource.
+     * @param mementoDateTime a RFC-1123 datetime
+     * @throws Exception
```

Comment:
JavaDoc wants a description on the Exception as well. Such as:
`@throws Exception on HTTP request error`